mergin/merginproject.py

```diff
@@ -169,7 +181,7 @@ def version(self) -> str:
         """Returns project version (e.g. "v123")"""
         if self._metadata is None:
             self._read_metadata()
-        return self._metadata["version"]
+        return self._metadata["version"] if self._metadata["version"] else "v0"
```

Comment:
have you verified it is still the case that "version" can be empty string when the project is freshly created? it would be good to double check whether we need this code... and if we do, I would suggest to have also e.g. `new_metadata_empty_project.json` for testing purposes to ensure this is working correctly...

Reply:
Seems it is "v0" now. Also added new metadata file for testing.